.github/workflows/proto-docker.yml

```diff
@@ -9,45 +9,47 @@ on:
     paths:
       - "contrib/devtools/Dockerfile"
 
-permissions:
-  contents: read
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: cosmos/proto-builder
+
+  # Allow one concurrent deployment
+concurrency:
+  group: "proto-docker"
+  cancel-in-progress: true
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
     steps:
       # set VERSION to new version when making changes, when merged to main the image will automatically be pushed
       - uses: actions/checkout@v3
-      - name: Prepare
-        id: prep
-        run: |
-          DOCKER_IMAGE=tendermintdev/sdk-proto-gen
-          VERSION=latest
-          TAGS="${DOCKER_IMAGE}:${VERSION}"
-          echo ::set-output name=tags::${TAGS}
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@master
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
         with:
-          platforms: all
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          # modify value when deploying a new version
+          tags: |
+            type=semver,pattern={{version}},value=v0.8.0 
```

Comment:
Can't we use `${VERSION}` instead of hard-coding it?

Reply:
Shouldn't we keep that logic then? And only update the github username https://github.com/cosmos/cosmos-sdk/commit/1cab2bc7f4572c07e47d7b5fe97bfd91183d6960

Reply:
we would modify it manually then push and change it back to latest. there isn't an automated way to run this

Reply:
nvm I know a way